source/lambda/documentprocessor/lambda_function.py

```diff
@@ -64,7 +64,7 @@ def processRequest(request):
     file_type = get_mime_type(request)
 
     # If not expected extension, change status to FAILED and exit
-    if(file_type and file_type not in ['application/pdf', 'image/png', 'image/jpeg']):
```

Comment:
I dont think this still solvess the case where file_type is None.
filetype.guess function can return None value. When filetype is None, it would not fall into this if condition (in fact it would not fall in any if condition) and hence would not throw an error

I think the condition should simply be 

``` if( file_type not in ['application/pdf', 'image/png', 'image/jpeg']):```